actions/army_control.py

```diff
@@ -190,10 +190,10 @@ def idle_unit(self, unit):
             and len(local_controller.zerglings.ready) < 41
             and local_controller.townhalls
             and self.retreat_units
-            and not local_controller.counter_attack_vs_flying
         ):
-            self.move_to_rallying_point(unit)
-            return True
+            if not local_controller.counter_attack_vs_flying:
```

Comment:
i dont understand what the difference here is? it does exactly the same, but with an additional `if` statement.

Reply:
its to fix a pylint error - too many booleans on one statement

Reply:
i start to dislike pylint more and more.